webapp-src/src/Profile/Navbar.js

```diff
@@ -1,4 +1,4 @@
-import React, { Component } from 'react';
+import React, { Component, useState, useEffect } from 'react';
```

Comment:
It looks like `useState` and `useEffect` are not used in the component, we should remove them from the import I guess

Reply:
Apologies — these are leftovers from an abandoned experiment regarding window resize events. They may be safely removed.